### ISSUE TITLE
0crat sends same messages to the ticket multiple times

### ISSUE THREAD

**Issue by sereshqua** [zerocracy/zerocracy.github.io#78]
**Steps to reproduce**:

1. Do a Quality Review and write `@0crat quality good` in the ticket

2. See same 0crat multiple messages about project NOT having enough funds:

[Here](https://github.com/Harvest-foundation/core/issues/428#issuecomment-478939031), [here (2nd)](https://github.com/Harvest-foundation/core/issues/428#issuecomment-478939738) [and here](https://github.com/Harvest-foundation/core/issues/428#issuecomment-478940933)

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by sereshqua** [comment]
@g4s8 i am sorry, i opened the issue in wrong repository (should have opened [here](https://github.com/zerocracy/farm/issues/))

**Comment by sereshqua** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@sereshqua yes, move it please

**Comment by sereshqua** [comment]
@g4s8 done - please [see](https://github.com/zerocracy/farm/issues/1999) 

can you please close this task as duplicate?

**Comment by g4s8** [comment]
@sereshqua no, I don't have write access to this repo, so I can't close issues
